addons/calendar/models/calendar_recurrence.py

```diff
@@ -104,13 +104,13 @@ class RecurrenceRule(models.Model):
     end_type = fields.Selection(END_TYPE_SELECTION, default='count')
     interval = fields.Integer(default=1)
     count = fields.Integer(default=1)
-    mo = fields.Boolean()
-    tu = fields.Boolean()
-    we = fields.Boolean()
-    th = fields.Boolean()
-    fr = fields.Boolean()
-    sa = fields.Boolean()
-    su = fields.Boolean()
+    mo = fields.Boolean(string='Monday')
+    tu = fields.Boolean(string='Tuesday')
+    we = fields.Boolean(string='Wednesday')
+    th = fields.Boolean(string='Thursday')
+    fr = fields.Boolean(string='Friday')
+    sa = fields.Boolean(string='Saturday')
+    su = fields.Boolean(string='Sunday')
```

Comment:
If you change the field name in stable, you have to update translations. I don't think it's necessary here. These are technical fields handled by the day of week widget.